From b8f16e1421fe0565e5103a25cf61abd9b4fd9a03 Mon Sep 17 00:00:00 2001
From: Zhangbin Tong <zebulun.tong@rock-chips.com>
Date: Thu, 16 Aug 2018 19:49:24 +0800
Subject: [PATCH] arm: dts: rk3128x: Change cpu opp-microvolt form one entry to
 three

Single entry is for target voltage and three entries are for
<target min max> voltages. Change cpu opp-microvolt form one entry to
three entries and set maximum acceptable voltage to a high value so that
regulator device can supply multiple consumers at the same time.

Change-Id: Id74c570afe702e879504a597a3d9fb3754125f2f
Signed-off-by: Zhangbin Tong <zebulun.tong@rock-chips.com>
---
 arch/arm/boot/dts/rk3128x.dtsi | 20 ++++++++++----------
 1 file changed, 10 insertions(+), 10 deletions(-)

diff --git a/arch/arm/boot/dts/rk3128x.dtsi b/arch/arm/boot/dts/rk3128x.dtsi
index bed1a10079bb..5b32f875b8e3 100644
--- a/arch/arm/boot/dts/rk3128x.dtsi
+++ b/arch/arm/boot/dts/rk3128x.dtsi
@@ -90,35 +90,35 @@
 
 		opp-408000000 {
 			opp-hz = /bits/ 64 <408000000>;
-			opp-microvolt = <950000>;
+			opp-microvolt = <950000 950000 1175000>;
 			opp-microvolt-L0 = <950000 950000 1175000>;
-			opp-microvolt-L1 = <950000 950000 1125000>;
+			opp-microvolt-L1 = <950000 950000 1175000>;
 			clock-latency-ns = <40000>;
 			opp-suspend;
 		};
 		opp-600000000 {
 			opp-hz = /bits/ 64 <600000000>;
-			opp-microvolt = <975000>;
+			opp-microvolt = <975000 975000 1175000>;
 			opp-microvolt-L0 = <975000 975000 1175000>;
-			opp-microvolt-L1 = <975000 975000 1125000>;
+			opp-microvolt-L1 = <975000 975000 1175000>;
 		};
 		opp-816000000 {
 			opp-hz = /bits/ 64 <816000000>;
-			opp-microvolt = <1000000>;
+			opp-microvolt = <1000000 1000000 1175000>;
 			opp-microvolt-L0 = <1000000 1000000 1175000>;
-			opp-microvolt-L1 = <1000000 1000000 1125000>;
+			opp-microvolt-L1 = <1000000 1000000 1175000>;
 		};
 		opp-1008000000 {
 			opp-hz = /bits/ 64 <1008000000>;
-			opp-microvolt = <1175000>;
+			opp-microvolt = <1175000 1175000 1175000>;
 			opp-microvolt-L0 = <1175000 1175000 1175000>;
-			opp-microvolt-L1 = <1125000 1125000 1125000>;
+			opp-microvolt-L1 = <1125000 1125000 1175000>;
 		};
 		opp-1200000000 {
 			opp-hz = /bits/ 64 <1200000000>;
-			opp-microvolt = <1175000>;
+			opp-microvolt = <1175000 1175000 1175000>;
 			opp-microvolt-L0 = <1175000 1175000 1175000>;
-			opp-microvolt-L1 = <1125000 1125000 1125000>;
+			opp-microvolt-L1 = <1125000 1125000 1175000>;
 		};
 	};
 
-- 
2.35.3

